h2o-genmodel/src/main/java/hex/genmodel/tools/PredictCsv.java

```diff
@@ -188,7 +192,9 @@ private void run() throws Exception {
         }
 
         output.write("\n");
+        lineNum++;
       }
+      reader.close();
```

Comment:
reverted back to use CsvReader;
remove the constructor;
move all close() function to finally part of try/catch/finally block